# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../palo/hotfix.patch
# Copyright (C) 2018 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- palo-2.00/palo/palo.c.vanilla	2018-08-16 18:20:19.241434704 +0000
+++ palo-2.00/palo/palo.c	2018-08-16 18:20:29.445434551 +0000
@@ -22,6 +22,7 @@
 #include <sys/mman.h>
 #include <sys/types.h>
 #include <sys/wait.h>
+#include <sys/stat.h>
 #if LONG_OPTIONS
 # define _GNU_SOURCE
 # include <getopt.h>
